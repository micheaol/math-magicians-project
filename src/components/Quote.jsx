import React from 'react';

const Quote = () => (
  <div className="quote-wrapper">
    <h3>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad dolore assumenda eligendi </h3>
  </div>
);

export default Quote;
