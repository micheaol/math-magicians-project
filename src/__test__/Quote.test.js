import { render, screen, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

afterEach(() => {
  cleanup();
});
describe('render quote to screen', () => {
  test('render quote component to screen', () => {
    render(<Quote />);
    const quoteElement = screen.getByTestId('quote-wrapper');
    expect(quoteElement).toBeInTheDocument();
  });

  test('matches snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
