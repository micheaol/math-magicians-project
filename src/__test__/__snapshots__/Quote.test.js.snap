// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render quote to screen matches snapshot 1`] = `
<div
  className="quote-wrapper"
  data-testid="quote-wrapper"
>
  <h3>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad dolore assumenda eligendi 
  </h3>
</div>
`;
